drivers/espi/espi_mchp_xec.c

```diff
@@ -0,0 +1,822 @@
+/*
+ * Copyright (c) 2019 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <kernel.h>
+#include <soc.h>
+#include <errno.h>
+#include <espi.h>
+#include "espi_utils.h"
+
+#define M2S             1ul
```

Comment:
Always prefix your macros with something that identify the owner so here ESPI_XEC_ for instance